uwblueprint/paramedics-react
# Jl ccp snackbar

[Notion ticket](https://www.notion.so/uwblueprintexecs/CCP-patient-snackbar-link-039b5a16d08740d1a5bba2433876498e)

## Changes
- made the "view patient details" button in the ccp snackbar that pops up after transporting a patient functional
- added a new route dedicated to the patient details dialog in the top level App file
- added a PatientDetailsDialogRouting wrapper that allows external access of the dialog outside of the patient table in ccp
- added a redirect to the edit page on this function in case link is revisited 

## Screenshots
![image](https://user-images.githubusercontent.com/44094521/94507289-66db9980-01dd-11eb-955d-07a9d7e289f7.png)
![image](https://user-images.githubusercontent.com/44094521/94508242-84116780-01df-11eb-8eee-46a4ced912f2.png)

## Testing
- navigate to the ccp dashboard
(http://localhost:3000/events/3/ccps/1 should have patients)
- click on a patient that has not been transported yet
- edit patient status to transport
- complete patient transport editing and the snackbar should pop up
- click "view patient details" in the snackbar. the patient details dialog should pop up
- make sure all actions in the dialog work properly
- reload route. you should be redirected to the patient editing page

## Checklist
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

before opening PR

filling out PR

after opening PR
